templates/node_parameter_table.html

```diff
@@ -321,7 +321,7 @@ <h5 class="bottomWindowHeader">Node Fields Table: <span>(No selection)</span></h
                                                         <!-- ko if: Setting.findValue(Setting.ALLOW_COMPONENT_EDITING) -->
                                                             <div class="dropdown parameterTableTypeCustomSelect">
                                                                 <a class="dropdown-toggle dropdown-control" data-bind="text:$data.defaultValue, disabled: ParameterTable.getNodeLockedState($data)" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></a>
-                                                                <div class="dropdown-menu dropdown-menu-right dropdown-area" aria-labelledby="parameterTableTpeCustomSelect">
+                                                                <div class="dropdown-menu dropdown-menu-right dropdown-area" aria-labelledby="parameterTableTpeCustomSelect" data-bind="event:{mousewheel: $root.slowScroll}">
```

Comment:
The two changes in this file are slightly different. One adds "clickBubble:false" and the other does not.

Should it be added to both? Or not required in either? Or is there a reason for having one and not the other?

Reply:
leftover from testing an issue i had, no longer needed.